yabt/buildcontext.py

```diff
@@ -128,11 +128,11 @@ def walk_target_deps_topological_order(self, target: Target):
 
     def generate_direct_deps(self, target: Target):
         """Generate only direct dependencies of `target`."""
-        yield from (self.targets[dep_name] for dep_name in target.deps)
+        yield from (self.targets[dep_name] for dep_name in sorted(target.deps))
```

Comment:
this does not feel to me the correct place to sort?

Reply:
oh sorrt, I take it back

Reply:
why not?

Reply:
> oh sorry, I take it back

oh, ok :-)